docs/configuration.rst

```diff
@@ -221,6 +221,16 @@ Panel options
   The SQL panel highlights queries that took more that this amount of time,
   in milliseconds, to execute.
 
+* ``SQL_TOKEN_GROUPING``
+
+  Default: ``True``
+
+  Panel: SQL
+
+  Controls SQL token grouping.
+  When set to ``True``, it might cause render slowdowns
+  when a view make long SQL textual queries.
```

Comment:
This part of the documentation should briefly explain what grouping of SQL tokens means so that the developer understands what this setting does when it's set to `True` besides the possible performance impact.